ui_tests/pages/inventory_page.py

```diff
@@ -0,0 +1,336 @@
+        confirm_btn.click()
+        time.sleep(1)
+
+    def set_community_string(self, community_string, edit=False):
+        logger.info(f"Set community string: {community_string}")
+        community_input_field_xpath = (
+            "//div[@data-test='sc4snmp:form:community-input']//span//input"
+        )
+        community_input_field = driver.find_element(
+            By.XPATH, community_input_field_xpath
+        )
+        if edit:
+            helper.clear_input(community_input_field)
+        community_input_field.send_keys(community_string)
+
+    def clik_edit_inventory_entry(self, host_ip):
+        logger.info(f"Edit entry from inventory list with: {host_ip}")
+        edit_inventory_entry_btn_xpath = f"//button[@data-test='sc4snmp:inventory-row-edit' and ancestor::tr//td[text()='{host_ip}']]"
+        edit_inventory_entry_btn = driver.find_element(
+            By.XPATH, edit_inventory_entry_btn_xpath
+        )
+        edit_inventory_entry_btn.click()
+
+    def get_edit_inventory_notice(self):
+        logger.info(f"Get edited inventory popup text")
+        edited_inventory_popup_text_xpath = f"//div[@data-test='modal']//div//p"
+        edited_inventory_popup_text = driver.find_element(
+            By.XPATH, edited_inventory_popup_text_xpath
+        )
+        return edited_inventory_popup_text.text
+
+    def select_group_inventory_type(self):
+        logger.info(f"Select group inventory type")
+        group_inventory_type_btn_xpath = (
+            f"//button[@data-test='sc4snmp:form:inventory-type-group']"
+        )
+        group_inventory_type_btn = driver.find_element(
+            By.XPATH, group_inventory_type_btn_xpath
+        )
+        group_inventory_type_btn.click()
+
+    def get_host_missing_error(self):
+        logger.info(f"Get host missing error")
+        host_error_xpath = f"//p[@data-test='sc4snmp:ip-group-error']"
+        try:
+            host_error = driver.find_element(By.XPATH, host_error_xpath)
+            return host_error.text
+        except selenium.common.exceptions.NoSuchElementException:
+            return None
+
+    def get_community_string_missing_error(self):
+        logger.info(f"Get community string missing error")
+        community_error_xpath = f"//p[@data-test='sc4snmp:community-error']"
+        try:
+            community_error = driver.find_element(By.XPATH, community_error_xpath)
+            return community_error.text
+        except selenium.common.exceptions.NoSuchElementException:
+            return None
+
+    def get_walk_invalid_value_error(self):
+        logger.info(f"Get walk interval invalid value error")
+        community_error_xpath = f"//p[@data-test='sc4snmp:walk-interval-error']"
+        try:
+            community_error = driver.find_element(By.XPATH, community_error_xpath)
+            return community_error.text
+        except selenium.common.exceptions.NoSuchElementException:
+            return None
```

Comment:
The same comment about similar functions.

Reply:
merged into `_get_error_for_missing_or_invalid_inventory_field(self, field)`